src/shrdlu/blockl.6

```diff
@@ -250,13 +250,15 @@
 (DEFUN PACKO
        (OBJ TYPE)
        (PROG (XX)
+	     (setq xx nil)
 	     (MAPC (FUNCTION (LAMBDA (X)
 				     (AND (THVAL (QUOTE (THGOAL (!IS $?X
 								     $E
 								     TYPE)))
 						 (LIST (LIST (QUOTE X) X)))
 					  (SETQ XX (PACKORD X (SIZE X) XX)))))
-		   OBJ)
+		   (cond ((atom obj) (list obj))
+			 (t obj)))
```

Comment:
SHRDLU has a function LISTIFY does this.

Reply:
I’ll check out listify and change to use it.